src/Config/InfectionConfig.php

```diff
@@ -207,16 +207,16 @@ private function getExcludes(): array
     }
 
     /**
-     * @return string|null
+     * @return string
```

Comment:
`string|null` should be :P

Reply:
Woops went a bit overboard with the fixing here.